ls1intum/Artemis
# Bugfix Guided Tour: Resize errors and positioning dots

### Checklist
- [x] Client: I added client tests (Jest) related to the features

### Description
This PR should fix several issues with the guided tour including https://github.com/ls1intum/Artemis/issues/943 and https://github.com/ls1intum/Artemis/issues/946 

- [x] The SourceTree setup tutorial video should be cropped and the noise in the audio should be removed
- [x] The tour should not be triggered on resize
- [x] The tour step should style should be responsive when resizing the browser window
- [x] The already executed step should hint should have a checkmark
- [x] The navigation dots on the tour step should be placed in the center.
- [x] The position of the dots should be calculated correctly when continuing the tour, see issue in screenshot:
![image](https://user-images.githubusercontent.com/33764166/66788907-f7d83880-eed8-11e9-837c-9f03c622e9d7.png)
- [x] The embedded video should not display related videos, see issue in screenshot: 
![image](https://user-images.githubusercontent.com/33764166/66827577-8c28b680-ef3e-11e9-8d1e-5ffbc1e48e5f.png)

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Overview Page
3. Tour should be displayed automatically when it has not been visited before
4. Close tour and resize browser window
5. Tour should not be started automatically
6. When navigating through the tour, the dots should be displayed correctly

### Screenshots
Tour with browser resizing
![1-min](https://user-images.githubusercontent.com/33764166/66864620-18f66300-ef85-11e9-94a5-129b362e48d5.gif)
Tour does not trigger on resize
![2](https://user-images.githubusercontent.com/33764166/66864419-a4232900-ef84-11e9-83d0-39ff3050d5cc.gif)
Retested success tour
![3](https://user-images.githubusercontent.com/33764166/66864474-c7e66f00-ef84-11e9-99f5-786632fc1a4b.gif)
Retested fail tour
![4](https://user-images.githubusercontent.com/33764166/66864500-d3399a80-ef84-11e9-94a3-bccf4572ebf2.gif)
No suggested videos when pausing or finishing the video
![image](https://user-images.githubusercontent.com/33764166/66865111-16e0d400-ef86-11e9-9191-cce3a799211f.png)
